src/Buildalyzer/Logging/EventProcessor.cs

```diff
@@ -89,9 +115,29 @@ private void ProjectStarted(object sender, ProjectStartedEventArgs e)
             _currentResult.Push(null);
         }
 
+        private string GetTargetFramework(ProjectStartedEventArgs e)
+        {
+            string normalizedProjectFilePath = AnalyzerManager.NormalizePath(e.ProjectFile);
+            string tfm = null;
+            if (!_targets.TryGetValue(normalizedProjectFilePath, out tfm))

```

Comment:
This part is confusing me. If I'm reading it correctly, we're caching the first TFM we see for each project file and always returning that as the TFM for that project file? That doesn't seem right... We want to always return the current project's TFM, right - that's why the `_results` key was changed to include TFM in the first place.